src/Symfony/Component/Messenger/Bridge/Amqp/Transport/Connection.php

```diff
@@ -318,7 +318,7 @@ public function countMessagesInQueues(): int
     private function publishWithDelay(string $body, array $headers, int $delay, ?AmqpStamp $amqpStamp = null): void
     {
         $routingKey = $this->getRoutingKeyForMessage($amqpStamp);
-        $isRetryAttempt = $amqpStamp ? $amqpStamp->isRetryAttempt() : false;
+        $isRetryAttempt = $amqpStamp && $amqpStamp->isRetryAttempt();
```

Comment:
```suggestion
        $isRetryAttempt = $amqpStamp?->isRetryAttempt();
```

Reply:
Hmm, this doesn't seem to work: https://github.com/symfony/symfony/actions/runs/10281717328/job/28451844143?pr=57868